tests/test_merging.py

```diff
@@ -96,3 +96,20 @@ def test_merge_qnodes_error():
 
     with pytest.raises(ValueError):
         merge_qnodes(qnodes1, qnodes2)
+
+
+def test_merge_knodes():
+    """Test merge_knodes"""
+    with open("tests/test_jsons/test_merge_knodes1.json", "r") as file:
+        knodes1 = json.load(file)
+    with open("tests/test_jsons/test_merge_knodes2.json", "r") as file:
+        knodes2 = json.load(file)
+
+    merged_knodes = merge_knodes(knodes1, knodes2)
+    with open("output.json", "w") as file:
+        json.dump(merged_knodes, file)
```

Comment:
Why do we need to write this result to disk?

Reply:
good catch! I was using that statement for debugging purposes. I will remove that.